config/servicemappings/compute.yaml

```diff
@@ -1298,6 +1298,45 @@ spec:
             kind: Project
             version: v1beta1
             group: resourcemanager.cnrm.cloud.google.com
+    - name: google_compute_network_firewall_policy_association
+      kind: ComputeNetworkFirewallPolicyAssociation
+      metadataMapping:
+        name: name
```

Comment:
Ack. I'll wait further instructions/

Reply:
Hi @katrielt are you open to add this resource as a v1alpha1 resource for now so that you can have a CRD to experiment with soon?

The disadvantage of v1alpha1 is that we don't guarantee backwards compatibility. But as the resource ID field here is not yet guaranteed to be required, unique and immutable, it may be good to start with having a v1alpha1 resource.

The current way to add a v1alpha1 CRD is through the autogen channel. Here is an example about how to add the service mapping under autogen folder and enable it via the allowlist: https://github.com/GoogleCloudPlatform/k8s-config-connector/compare/master...maqiuyujoyce:202311-v1alpha1-crd.

Reply:
As discussed offline, `name` should be a reasonable resource ID.